requirements.txt

```diff
@@ -8,4 +8,9 @@ pandas
 tqdm
 pypet
 deap
-dill
\ No newline at end of file
+dill
+symengine
+sympy
```

Comment:
for symbolic dynamics definition + translating symbolics to string for `numba` backend